manager/model/collector.go

```diff
@@ -0,0 +1,73 @@
+package model
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+const database = "alba"
+const collector = "collector"
+const uri = "mongodb://root:example@mongo:27017"
+
+//Collector represents the information needed for frequent data collection operation
+type Collector struct {
+	ID                 string    `bson:"id"`                   // Initials entity like 'trt13'.
+	Entity             string    `bson:"entity"`               // Entity from which the collector extracts data like 'Tribunal Regional do Trabalho 13° Região'.
+	City               string    `bson:"city"`                 // City of the entity from which the collector extracts data.
+	FU                 string    `bson:"fu"`                   // Federation unit of the entity from which the collector extracts data.
+	UpdateDate         time.Time `bson:"update_date"`          // Update data of the collector register.
+	Path               string    `bson:"path"`                 // Collector repository path. Using the import pattern in golang like 'github.com/dadosjusbr/coletores/tree/master/trt13'.
+	Frequency          int       `bson:"frequency"`            // Frequency of the collector execution in days. Values must be between 1 and 30. To be executed monthly it must be filled with '30'.
```

Comment:
Go tem uma tipo de dados para isso, `time.Duration`. Assim, usuário poderá usar strings como `24h`. Mais em [ParseDuration](https://golang.org/pkg/time/#ParseDuration). O que acha?

Reply:
Acho que fica menos intuitivo para a pessoa usuária ter que passar de dias para horas, pensando a frequência atual apenas como dias, como falamos na reunião que por default seria 30 (mensal).
E se a gente for utilizar a [estutura do cron](https://godoc.org/github.com/robfig/cron) talvez não tenhamos ganho com a conversão.

Reply:
Talvez possamos decidir melhor quando implementarmos a função para executar periodicamente, pois aí saberemos o formato necessário. O que acha?

Reply:
Beleza.